runtime_builders/publish_builders.py

```diff
@@ -22,6 +22,7 @@
 import sys
 
 import builder_util
+from verify_manifest import verify_manifest
```

Comment:
We typically import just the package and then reference stuff like verify_manifest.verify_manifest

Reply:
fixed